server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CICSParser.g4

```diff
@@ -415,14 +417,14 @@ cics_handle_abend: ABEND (CANCEL | PROGRAM cics_name | LABEL cics_label | RESET
 cics_handle_aid: AID (ANYKEY (cics_label)? | CLEAR (cics_label)? | CLRPARTN (cics_label)? | ENTER (cics_label)? |
                  LIGHTPEN (cics_label)? | OPERID  (cics_label)? | pa_option (cics_label)? | pf_option (cics_label)? |
                  TRIGGER  (cics_label)? | cics_resp)*;
-cics_handle_condition: CONDITION (mama cics_label? | cics_resp)+;
+cics_handle_condition: CONDITION ((cics_conditions | mama) cics_label? | cics_resp)+;
```

Comment:
May be simplified to `(cics_conditions | mama | cics_label | cics_resp)+`

Reply:
I'd like not to do that.
```
cics_label: LPARENCHAR paragraphNameUsage RPARENCHAR;
```
So the following line will be valid `EXEC CICS HANDLE CONDITION (10-EXIT) END-EXEC.`

Reply:
But this one is valid, isn't it?

Reply:
Sorry, you are right